internal/network/manifests_generator.go

```diff
@@ -315,7 +315,7 @@ data:
 	prodServiceBaseURL  = "https://api.openshift.com"
 	stageServiceBaseURL = "https://api.stage.openshift.com"
 	stageTelemeterURL   = "https://infogw.api.stage.openshift.com"
-	dummyURL            = "https://dummy.com"
+	dummyURL            = "https://dummy.example"
```

Comment:
I liked the suggestion of `.invalid` for this.
Why change to `.example`?

Reply:
I don't have a particular preference, but it's more of a convention - it's a default value present if the user does not set anything explicitly. In such a scenario it feels somehow not natural that we by default put an invalid value - I see it a bit like default "-1" in a field asking for my age.

But it's not a strong preference, we can go with `dummy.invalid`

Reply:
I suppose that depends on the case when we use this `dummyURL` is this something we expect users to update manually?
I thought it was something we explicitly expected to be unused.

If it's the former then, yeah, example makes more sense. @ybettan ?

Reply:
Ok, changed to `.invalid`